data-transfer-objects/src/main/java/eu/arrowhead/dto/OrchestrationRequestDTO.java

```diff
@@ -0,0 +1,12 @@
+package eu.arrowhead.dto;
+
+import java.util.List;
+import java.util.Map;
+
+public record OrchestrationRequestDTO(
+		OrchestrationServiceRequirementDTO serviceRequirement,
+		List<String> orchestrationFlags,
```

Comment:
should be Map<String,Boolean>

Reply:
Don't agree :)
I think the List is more straight-forward.

Reply:
The word flags in this context indicates to me there are some booleans involves. Also, consumers should have the option to tell us explicitly they don't want something, for example allow_translation is false. It can improve the readability of the consumer code.